example_governance_doc.md

```diff
@@ -85,22 +122,23 @@ You should engage the SWG as early as possible if any of these match the schema
 * Deprecations of fields or subgraphs ([see the below deprecations section](#deprecations))
 * If a review would be helpful
 
-
 #### How to Engage
 
 While schema reviews are helpful, the core graph team also realizes that meeting after meeting isn't conducive to productivity. 
 
-With that, the SWG strongly recommends schema reviews whenever possibly by doing async reviews on a shared Google Doc. A template for such is [here](). This document will ask for necessary context, such as the currently proposed schema, alternative schemas, and any UI mocks if possible.
+With that, the SWG strongly recommends schema reviews using Apollo's [schema proposals](https://www.apollographql.com/docs/graphos/platform/schema-management/proposals) feature. Managing schema changes using schema proposals enables teams to propose schema changes that are validated against our development variant, running composition, linting and schema checks on every change.
 
-To engage with the SWG, please reach out to #graphql-swg on Slack along with the above doc. 
+To engage with the SWG, please reach out to #graphql-swg on Slack along with a link to the appropriate schema proposal. 
 
 #### Next Steps
 
-Once the SWG has received the document, they will begin to provide comments and suggestions within the doc. 
+Once the SWG has received the proposal, they will begin to provide comments and suggestions within the tool. 
 
 For new subgraphs, there will also be a separate meeting to discuss the proposal in-depth and provide further feedback. 
 
-Once both the SWG and subgraph team(s) agree on the change, the proposal will be considered "approved" and will move to be implemented into the supergraph. 
+Once both the SWG and subgraph team(s) agree on the change, the proposal will be considered "approved" and will move to be implemented into the supergraph.
+
+Subgraph teams can pull down the approved proposal using `rover` and once published to the supergraph the proposal will be marked as "implemented"
```

Comment:
Might be worth having an example here?

Reply:
What did you have in mind? I wan't to avoid duplicating instructions from the docs link here